apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -133,27 +136,21 @@ export default {
     }
   },
   computed: {
-    ...mapGetters('Files', ['selectedFiles', 'shareOpen', 'shares', 'sharesError', 'sharesLoading']),
+    ...mapGetters('Files', ['selectedFiles', 'shareOpen', 'shares', 'sharesError', 'sharesLoading', 'davProperties']),
     ...mapState(['user']),
     $_ocCollaborationStatus_autocompletePlacholder () {
       return this.$gettext('Add name(s), email(s) or federation ID\'s')
     },
     owner () {
-      if (this.shares.length === 0) {
-        return {
-          avatar: 'https://picsum.photos/64/64?image=1074',
-          name: this.user.id,
-          displayName: this.user.displayname
-        }
-      }
-      const s = this.shares[0]
-      return {
-        avatar: 'https://picsum.photos/64/64?image=1074',
-        name: s.uid_owner,
-        displayName: s.info.displayname_owner
-      }
-    }
+      this.$client.files.fileInfo(this.selectedFiles[0].path, this.davProperties).then(res => {
+        let ownerId = res['fileInfo']['{http://owncloud.org/ns}owner-id']
+        this.ownerAvatar = `${this.$root.config.server}/remote.php/dav/avatars/${ownerId}/128.png`,
```

Comment:
This will only work by accident due to the cookie which is saved at login within owncloud.
All API calls need to be done use the bearer header and the stored access token (oauth2/openid connect).